packages/admin/src/admin.ts

```diff
@@ -0,0 +1,197 @@
+import { Context } from '@textile/context'
+import {
+  CopyAuthOptions,
+  GrpcAuthentication,
+  WithUserAuthOptions,
+} from '@textile/grpc-authentication'
+import * as api from './api'
+
+export class Admin extends GrpcAuthentication {
```

Comment:
The actual admin class, following @andrewxhill 's pattern for this. The name is up for debate!

Reply:
nice! might as well i guess, though the benefit was mostly for getting nice docs, which wouldn't matter here.

Reply:
Yeh, it was just a nice pattern to use because it actually makes the functions themselves a lot more reusable.